packages/@react-spectrum/table/docs/TableView.mdx

```diff
@@ -511,7 +517,7 @@ function AsyncSortTable() {
         items: items.sort((a, b) => {
           let first = a[sortDescriptor.column];
           let second = b[sortDescriptor.column];
-          let cmp = (parseInt(first) || first) < (parseInt(second) || second) ? -1 : 1;
+          let cmp = collator.compare((parseInt(first) || first), (parseInt(second) || second));
```

Comment:
Mind removing the `parseInt`s? We don't need them anymore since we are doing string comparisons now via `collator.compare`
```suggestion
          let cmp = collator.compare(first, second);
```